Master mdl 71679 mod quiz 52

safatshahin opened the pull request:
PR for #52 

Please dont merge if the PR is alright, instead make a comment so that I can squash the commits in this branch.

guillogo commented:
Hi @safatshahin,

Thanks for working on this, what do you think about adding the deprecated functions in the upgrade.txt in mod_quiz?. If you don't see this necessary, it's good to go.

Tested:
- User test (Done detailed tests as this PR removes all classes in question/classes/bank as are not used anymore).
- Unit test OK.
- Behat test mod_quiz OK.

Gracias!

safatshahin commented:
Hi @guillogo 
Thanks for your review of this. As discussed, kept the upgrade text as it is to avoid duplication. Squashed the commits and added to the base branch.
Cheers!
